packages/graphql/src/environment.ts

```diff
@@ -17,32 +17,9 @@
  * limitations under the License.
  */
 
-/*  Getters: 
-    https://stackoverflow.com/questions/45194598/using-process-env-in-typescript
-    https://dev.to/isthatcentered/typing-process-env-and-dealing-with-nodeenv-3ilm
-    If you just do;
-        exports const NEO4J_GRAPHQL_JWT_ROLES_OBJECT_PATH = process.env.NEO4J_GRAPHQL_JWT_ROLES_OBJECT_PATH;
-    the const will be undefined.
-*/
 const environment = {
-    get NEO4J_GRAPHQL_JWT_SECRET() {
-        return process.env.NEO4J_GRAPHQL_JWT_SECRET;
-    },
-    get NEO4J_GRAPHQL_JWT_NO_VERIFY() {
-        return Boolean(process.env.NEO4J_GRAPHQL_JWT_NO_VERIFY || false);
-    },
-    get NEO4J_GRAPHQL_ENABLE_REGEX() {
-        return Boolean(process.env.NEO4J_GRAPHQL_ENABLE_REGEX || false);
-    },
-    get NEO4J_GRAPHQL_JWT_ROLES_OBJECT_PATH() {
-        return process.env.NEO4J_GRAPHQL_JWT_ROLES_OBJECT_PATH;
-    },
-    get NPM_PACKAGE_VERSION() {
-        return process.env.NPM_PACKAGE_VERSION as string;
-    },
-    get NPM_PACKAGE_NAME() {
-        return process.env.NPM_PACKAGE_NAME as string;
-    },
+    NPM_PACKAGE_VERSION: process.env.NPM_PACKAGE_VERSION as string,
```

Comment:
Devil's advocate... Is there even any point in this file anymore?

It's far easier to just do `process.env.NPM_PACKAGE_VERSION` as opposed to:

```
import environment from "./environment";
...
environment.NPM_PACKAGE_VERSION;
```

Reply:
I'm keeping this its nice to have envs in one place.